Signal/src/view controllers/MessagesViewController.m

```diff
@@ -1211,7 +1316,11 @@ - (void)showConversationSettings
         DDLogDebug(@"%@ Ignoring request to show conversation settings, since user left group", self.tag);
         return;
     }
-    [self performSegueWithIdentifier:OWSMessagesViewControllerSeguePushConversationSettings sender:self];
+    
+    OWSConversationSettingsTableViewController *settingsVC = [[UIStoryboard storyboardWithName:AppDelegateStoryboardMain bundle:NULL]
+                                                              instantiateViewControllerWithIdentifier:@"OWSConversationSettingsTableViewController"];
+    [settingsVC configureWithThread:self.thread];
+    [self.navigationController pushViewController:settingsVC animated:YES];
```

Comment:
To use the customized text view, I had to switch to doing programmatic segues between these views. =/